modules/control/proto/control_cmd.proto

```diff
@@ -147,6 +153,8 @@ message SimpleLateralDebug {
   // term for control usage
   optional double lateral_error_feedback = 28;
   optional double heading_error_feedback = 29;
+
+  optional CurrentTargetPoint current_target_point = 30;
```

Comment:
I think you can just reuse the type defined in modules/common.

Reply:
Fixed. Thank you.